packages/manifold/contracts/gachaclaims/IERC1155GachaLazyClaim.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/// @author: manifold.xyz
+
+import "./IGachaLazyClaim.sol";
+
+/**
+ * Gacha 1155 Lazy Claim interface
+ */
+interface IERC1155GachaLazyClaim is IGachaLazyClaim {
+  struct Claim {
+    StorageProtocol storageProtocol;
+    uint32 total;
+    uint32 totalMax;
+    uint48 startDate;
+    uint48 endDate;
+    uint112 startingTokenId;
+    uint8 itemVariations;
+    string location;
+    address payable paymentReceiver;
+    address erc20;
+    uint256 cost;
```

Comment:
Make cost a uint96, and move it above erc20.  This way you can fit paymentReceiver and cost in one storage slot

Reply:
address is 160